src/main/deltachat/chat.ts

```diff
@@ -16,24 +19,24 @@ module.exports = class DCChat extends SplitOut {
     )
   }
 
-  getEncryptionInfo(contactId) {
+  getEncryptionInfo(contactId: number) {
     return this._dc.getContactEncryptionInfo(contactId)
   }
 
   getQrCode(chatId = 0) {
     return this._dc.getSecurejoinQrCode(chatId)
   }
 
-  leaveGroup(chatId) {
+  leaveGroup(chatId: number) {
     log.debug(`action - leaving chat ${chatId}`)
     this._dc.removeContactFromChat(chatId, C.DC_CONTACT_ID_SELF)
   }
 
-  setName(chatId, name) {
+  setName(chatId: number, name: any) {
```

Comment:
we know that the name is a string don't we?
If it also can be null, which it  probably can **not** in this case, you could do `string|null` in order to be supper correct and to comply with the compiler if we would ever activate "no Implicit null"

Reply:
We should first specify this in deltachat-node and then bubble it up.

Reply:
feel free to do so